taar/recommenders/local_recommender.py

```diff
@@ -46,6 +46,8 @@ def can_recommend(self, client_data):
 
     def recommend(self, client_data, limit):
         client_locale = client_data.get('settings.locale')
-        top_n_dict = self._load_model()
-        # TODO: do we want to truncate this?
-        return top_n_dict[client_locale]
+        top_n_dict = self.model()
+        top_n_list = top_n_dict[client_locale]
```

Comment:
What about making the lines from 49 to 53 just be:

`return self.model[client_locale][:limit]`